docs/operator-docs/lifecycle-policies.md

```diff
@@ -0,0 +1,152 @@
+---
+id: lifecycle-policies
+title: Lifecycle Policies
+hide_title: true
+---
+
+# Setting Lifecycle Policies
+
+This document will outline, in brief, how to set data lifecycle policies so that you are more easily able to control data storage costs while keeping your data accessible.
```

Comment:
I might put the Object Lifecycle management link up here to help explain what a lifecycle policy is.

Reply:
I added a reference to the 'more information' section at the end.  Given Jacob's commentary on the focus of the documents, I wanted to avoid highlighting information that's extraneous to the task at hand at the top of the document if possible.